app/src/main/java/org/simple/clinic/bloodsugar/selection/type/BloodSugarTypeAdapter.kt

```diff
@@ -35,13 +37,16 @@ class BloodSugarTypeAdapter(
   }
 }
 
-class BloodSugarTypeViewHolder(override val containerView: View) : RecyclerView.ViewHolder(containerView), LayoutContainer {
+class BloodSugarTypeViewHolder(
+    override val containerView: View,
+    val binding: ListBloodSugarTypeBinding
+) : RecyclerView.ViewHolder(containerView), LayoutContainer {
```

Comment:
Since we are no longer using the Kotlin synthetic view accessors for the adapter, we can drop the `LayoutContainer` implementation here.